src/components/PostTable.js

```diff
@@ -1,13 +1,125 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const TableWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 12px;
+  padding-bottom: 127px;
+`;
+
+const PostsTable = styled.table`
+  max-width: 786px;
+  width: 100%;
+  margin: -2 auto 0;
+  cell-collapse: 0;
+  border-spacing: 0;
+  border: 1px solid ${({ theme }) => theme.color.tableBorder};
+`;
+
+const TableHeading = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  max-width: 786px;
+  width: 100%;
+  margin: 0 auto;
+`;
+
+const Th = styled.th`
+  text-align: left;
+  font-size: ${({ theme }) => theme.font.size.small};
+  color: ${({ theme }) => theme.color.secondary};
+  border: 1px solid ${({ theme }) => theme.color.tableBorder};
+  padding: 8px 12px;
+  white-space: nowrap;
+`;
+
+const Td = styled.td`
+  border: 1px solid ${({ theme }) => theme.color.tableBorder};
+  padding: 8px 12px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  color: ${({ theme }) => theme.color.secondary};
+  font-size: ${({ theme }) => theme.font.size.small};
+`;
+
+const TitleTd = styled(Td)`
+  max-width: 350px;
+`;
+
+const AuthorTd = styled(Td)`
+  max-width: 104px;
+`;
+
+const TableLink = styled(Link)`
+  text-decoration: none;
+  color: ${({ theme }) => theme.color.externalLink};
+  font-size: ${({ theme }) => theme.font.size.small};
+`;
+
+const getTimePosted = (createdUTC) => {
+  const date = new Date(createdUTC * 1000);
+  const hourIn24 = date.getHours();
+  const hourIn12 = ((hourIn24 + 11) % 12) + 1;
```

Comment:
The calculation is not easy to follow but thanks to the variable naming it's immediately clear what's going on :+1: 

Fyi: I had a look how I implemented it since I had difficulties understanding the calculation :smile: Mine looks like this

```
const hour12hNotation = hour24hNotation % 12 || 12
```

Reply:
ohh clever!